components/flag.js

```diff
@@ -25,31 +25,41 @@ var supportedCountryCodes = [
   'yt', 'za', 'zm', 'zw']
 
 const FlagImg = styled.img`
-  width: 60px;
-  height: 60px;
+  width: ${props => props.size}px;
+  height: ${props => props.size}px;
   clip-path: circle(50% at 50% 50%);
 `
 
 const FlagContainer = styled.div`
   border-radius: 50%;
   padding-left: 3px;
   padding-top: 3px;
-  width: 66px;
-  height: 66px;
+  width: ${props => props.size + 6}px;
+  height: ${props => props.size + 6}px;
 `
 
-export const Flag = ({countryCode, center, border}) => {
+export const Flag = ({countryCode, size}) => {
   countryCode = countryCode.toLowerCase()
   if (supportedCountryCodes.indexOf(countryCode) === -1) {
     // Map unsupported country codes to ZZ
     countryCode = 'zz'
   }
   const src = `/static/flags/1x1/${countryCode}.svg`
   return (
-    <FlagContainer className='country-flag' center={center} border={border}>
-      <FlagImg src={src} />
+    <FlagContainer className='country-flag' size={size}>
+      <FlagImg src={src} size={size} />
     </FlagContainer>
   )
 }
 
+Flag.propTypes = {
+  countryCode: PropTypes.string.isRequired,
+  size: PropTypes.number,
```

Comment:
This should probably also be an `isRequired`

Reply:
:+1: